node_modules/oae-email/lib/api.js

```diff
@@ -383,6 +383,11 @@ var sendEmail = module.exports.sendEmail = function(templateModule, templateId,
 
     if (renderedTemplates['html']) {
         emailInfo.html = renderedTemplates['html'];
+
+        // We need to escape the &apos; entity because some e-mail clients
+        // don't (yet) support html5 rendering, such as Outlook
+        // Tip from http://stackoverflow.com/questions/419718/html-apostrophe
+        emailInfo.html = emailInfo.html.replace(/&apos;/g, '&#39;');
```

Comment:
Won't this just replace the first instance of `&apos;` but not the following ones?